pynest/nest/lib/hl_api_connections.py

```diff
@@ -350,6 +350,7 @@ def Connect(pre, post, conn_spec=None, syn_spec=None, model=None):
 
 
 @check_stack
+@deprecated("", "DataConnect: use Connect() with one_to_one rule")
```

Comment:
Better: `@deprecated('DataConnect', 'Connect() with one_to_one rule')`. In that way, the message will automatically  have standard format.